app/src/organisms/RunPreview/CommandIcon.tsx

```diff
@@ -30,11 +31,6 @@ export function CommandIcon(props: CommandIconProps): JSX.Element | null {
   }
 
   return iconName != null ? (
-    <Icon
-      {...styleProps}
-      name={iconName}
-      size={SPACING.spacing20}
-      flex="0 0 auto"
-    />
+    <Icon {...styleProps} size={svgWidth} name={iconName} flex="0 0 auto" />
```

Comment:
can we rename the `svgWidth` prop to `size`? Since both `size` and `svgWidth` props exist, it is a bit confusing. 
```suggestion
    <Icon {...styleProps} size={size} name={iconName} flex="0 0 auto" />
```